unstructured/ingest/v2/README2.md

```diff
@@ -0,0 +1,353 @@
+# Developing V2 Connectors
+## Intro
+The Unstructured open source repo processes documents (artifacts) in a pipeline. The Source and Destination connectors sit at the front and back of the pipeline. For a visual example see the flow diagram at the bottom (link to bottom).
+
+## Simplest Example of a Pipeline
```

Comment:
Minor: Perhaps "basic" instead of "simple?" Something that's simple to us might not be perceived as simple to others.